javascript/javascript2/week1/hyf-bay/main.js

```diff
@@ -0,0 +1,54 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+
+//function get color is to get a random color
+function getColor() {
+  let rgb1 = Math.floor(Math.random() * 256) + 1;
+  let rgb2 = Math.floor(Math.random() * 256) + 1;
+  let rgb3 = Math.floor(Math.random() * 256) + 1;
+  return "color : rgb(" + rgb1 + "," + rgb2 + "," + rgb3 + ")";
+}
+
+function getRating(ratingValue) {
+  let ratingStar = "";
+  for (let i = 0; i < ratingValue; i++) {
+    ratingStar += "*";
+  }
+  return ratingValue + " : " + ratingStar;
+}
+
+function renderProducts(products) {
+  // your code here
+  const main = document.querySelector("main");
+  main.setAttribute("style", "margin-left:3vw"); // left margin
+
+  const newul = document.createElement("ul");
+  main.appendChild(newul);
+
+  const ul = document.querySelector("ul");
+  products.forEach((product) => {
+    const li = document.createElement("li");
+
+    const h2 = document.createElement("h2");
+    h2.style.cssText = "text-decoration:underline;" + getColor();
+    //h2.setAttribute("style", getColor());
+
+    h2.innerHTML = product.name;
+    li.appendChild(h2);
+
+    const span = document.createElement("span");
+    span.setAttribute("style", getColor());
+    span.innerHTML = "<b>.</b>price: " + product.price;
+    li.appendChild(span);
```

Comment:
Can you remove list decoration for price?

Reply:
yes i can